test-docker-compose/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3'
```

Comment:
Could these files be inside `integration`?

Reply:
What do you mean? The `docker-compose` config is used for both integration tests and normal tests

Reply:
Okay.

I wonder if we could port the normal test functionality into integration tests, at least for those that depend upon a Grafana instance, as otherwise they're not really unit tests.

Reply:
Agreed. Integration tests are all we need, IMO. It tests the functionality end-to-end

Reply:
Let's move the whole thing back to the integration dir, when and if we start only using these instances there. Does that work?

Reply:
sure!